src/components/timetable-duration/timetable-duration.scss

```diff
@@ -1,5 +1,8 @@
 @use '../core/styles' as sbb;
 
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
```

Comment:
Same as link

Reply:
as there was no display property before migration, I thought to not add one now. for the box-sizing I think it's important that every component uses it to have a good consistency. However to be also consistent with display, I added inline-block to this component :)